src/js/tools/superselect.js

```diff
@@ -120,6 +119,27 @@ define(function (require, exports, module) {
     };
 
     /**
+     * Vector Mode requires that do not swallow pointer commands like we do in normal superselect
+     * @private
+     * @return {Array.<PointerEventPolicy>}
+     */
+    var _pointerPolicyList = function () {
+        var toolStore = this.flux.store("tool"),
+            vectorMode = toolStore.getVectorMode(),
+            pointerPolicy = null;
```

Comment:
No need to initialize this to null since you're going to overwrite immediately in either branch. Just a declaration will do fine here.